src/Microsoft.AspNet.Mvc.Core/ModelBinding/MutableObjectModelBinder.cs

```diff
@@ -555,9 +555,8 @@ internal void ProcessDto(ModelBindingContext bindingContext, ComplexModelDto dto
             // Determine T if this is an ICollection<T> property. No need for a T[] case because CanUpdateProperty()
             // ensures property is either settable or not an array. Underlying assumption is that CanUpdateProperty()
             // and SetProperty() are overridden together.
-            var collectionTypeArguments = propertyExplorer.ModelType
-                .ExtractGenericInterface(typeof(ICollection<>))
-                ?.GenericTypeArguments;
+            var collectionTypeArguments = TypeHelper.ExtractGenericInterface(
+                propertyExplorer.ModelType, typeof(ICollection<>))?.GenericTypeArguments;
```

Comment:
previous wrapping was much easier to grok and you're now putting multiple arguments on the same line after a break